lib/reader.js

```diff
@@ -46,14 +46,15 @@ class ParquetCursor  {
     this.columnList = columnList;
     this.rowGroup = [];
     this.rowGroupIndex = 0;
+    this.cursorIndex = 0;
```

Comment:
Should this also be set in the `rewind` function? https://github.com/ZJONSSON/parquetjs/pull/55/files#diff-e940db824fd38ce6d7405b4c6c09dda96482eb01cc7b1f453b24235795b85a2cR77